docs/figures/contract-invoke.excalidraw

```diff
@@ -0,0 +1,1412 @@
+        [
+          0,
+          0
+        ],
+        [
+          -151,
+          -0.2317888123496914
+        ]
+      ]
+    },
+    {
+      "type": "text",
+      "version": 82,
+      "versionNonce": 797369946,
+      "isDeleted": false,
+      "id": "X4xqFoDQrmh8exx1bv38C",
+      "fillStyle": "solid",
+      "strokeWidth": 1,
+      "strokeStyle": "solid",
+      "roughness": 0,
+      "opacity": 100,
+      "angle": 0,
+      "x": 898.5,
+      "y": 508.4,
+      "strokeColor": "#000000",
+      "backgroundColor": "#ffffff",
+      "width": 75,
+      "height": 19.2,
+      "seed": 370207130,
+      "groupIds": [],
+      "roundness": null,
+      "boundElements": null,
+      "updated": 1681333899559,
+      "link": null,
+      "locked": false,
+      "fontSize": 16,
+      "fontFamily": 3,
+      "text": "2. cache",
+      "textAlign": "center",
+      "verticalAlign": "top",
+      "containerId": null,
+      "originalText": "2. cache",
+      "lineHeight": 1.2,
+      "baseline": 15
+    },
+    {
+      "id": "LX_3czi9U2yRT874FbFJG",
+      "type": "text",
+      "x": 898.328125,
+      "y": 133,
+      "width": 309.6953125,
+      "height": 32.199999999999996,
+      "angle": 0,
+      "strokeColor": "#000000",
+      "backgroundColor": "#ffffff",
+      "fillStyle": "solid",
+      "strokeWidth": 1,
+      "strokeStyle": "solid",
+      "roughness": 0,
+      "opacity": 100,
+      "groupIds": [],
+      "roundness": null,
+      "seed": 2102201114,
+      "version": 151,
+      "versionNonce": 121423642,
+      "isDeleted": false,
+      "boundElements": null,
+      "updated": 1681337178806,
+      "link": null,
+      "locked": false,
+      "text": "PDO Contract Invocation",
+      "fontSize": 28,
+      "fontFamily": 2,
+      "textAlign": "center",
+      "verticalAlign": "top",
+      "baseline": 25,
+      "containerId": null,
+      "originalText": "PDO Contract Invocation",
+      "lineHeight": 1.15
+    },
+    {
+      "id": "tDjSdJs_JRj-2--sn4Adf",
+      "type": "text",
+      "x": 1020.6320190429688,
+      "y": 566.5,
+      "width": 66.7359619140625,
+      "height": 20,
+      "angle": 0,
+      "strokeColor": "#000000",
+      "backgroundColor": "#ffffff",
+      "fillStyle": "solid",
+      "strokeWidth": 1,
+      "strokeStyle": "solid",
+      "roughness": 0,
+      "opacity": 100,
+      "groupIds": [],
+      "roundness": null,
+      "seed": 58879686,
+      "version": 48,
+      "versionNonce": 1544690330,
+      "isDeleted": false,
+      "boundElements": null,
+      "updated": 1681337199465,
+      "link": null,
+      "locked": false,
+      "text": "end user",
+      "fontSize": 16,
+      "fontFamily": 1,
+      "textAlign": "center",
+      "verticalAlign": "top",
+      "baseline": 14,
+      "containerId": null,
+      "originalText": "end user",
+      "lineHeight": 1.25
+    }
+  ],
+  "appState": {
+    "gridSize": null,
+    "viewBackgroundColor": "#ffffff"
+  },
+  "files": {}
+}
```

Comment:
is there a particular reason to include this? have you checked license implications?

Reply:
Is Excalidraw a common tool? I've never heard of it. Any reason not to make them something more commonly used (google draw?). Or maybe i'm just way behind the times.

Reply:
I think excalidraw is actually being used under the hood by other whiteboard applications, I just realized recently that there's a separate in-browser service as well.